styles/style.css

```diff
@@ -0,0 +1,270 @@
+.container {
+  margin: 0 auto;
+  max-width: 1303px;
+}
+
+body {
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.bg-red {
+  background-color: #f72210;
+}
+
+header {
+  color: #fff;
+}
+
+.btn-primary {
+  background-color: #fff;
+  color: #f72210;
+  width: 150px;
+}
+
+.list-bk-color {
+  background-color: #fff;
+  padding: 0;
+}
+
+.nav-item {
+  margin: 0 auto;
+}
+
+.list-bk-color li {
+  list-style: none;
+}
+
+.search-input {
+  background-image: url('https://g.newsweek.com/www/images/icon-search-glass.svg');
+  background-repeat: no-repeat;
+  background-position: 100% 50%;
+  background-size: 24px 24px;
+  height: 32px;
+  border: none;
+  width: 90px;
+}
```

Comment:
Remember the purpose of the project is to use Bootstrap. Please favor the use of bootstrap classes in this case.